src/database/migrations/0004-create-application_forms.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE application_forms (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  opportunity_id INTEGER NOT NULL,
+  version INTEGER NOT NULL DEFAULT 1,
```

Comment:
I'm not sure we should actually define a default here

Reply:
When combined with the unique constraint below it means that the version is optional on the first (majority case) post but required on the second and following versions of that same opportunity. It seems like a slightly annoying burden to require a version for the first version posted. But it is probably simpler and clearer to require it every time. Requiring it every time tells the caller to be aware that multiple versions are supported. Then again, just because you have fancy options available (in this case versions of opportunities) does not mean we need to burden the simple case (a single opportunity) with the complexity of the complex case (multiple opportunities). In short, there is good reason to do it either way, I'm up for either.